src/content-scripts/empire-tracking/trackOnIngamePages.ts

```diff
@@ -248,7 +248,10 @@ function updateFleetTracking() {
         const destinationCoordinatesType = getCoordinateType(row.querySelector('.destFleet') ?? _throw('no dest icon found'));;
         const destinationCoordinates = parseCoordinates(destinationCoordinatesText.trim(), destinationCoordinatesType);
 
-        const tooltipHtml = row.querySelector('[class*="icon_movement"] .tooltip')?.getAttribute('title') ?? _throw('no tooltip found');
+        const tooltipElement = row.querySelector('[class*="icon_movement"] .tooltip, [class*="icon_movement_reserve"] .tooltip');
+        // Checking for data-tooltip-title and title as OGLight is refactoring it to title

```

Comment:
Oz confirmed me this will not be the case anymore in a later version of OGLight

Reply:
we could probably remove the whole fleet tracking anyways. I added it when I wanted to add a point distribution overview to the tracker, but it was bugged and other things were prioritized, so I didn't bother going at it again.